### DISCUSSION TITLE
How can I add a domain path prefix?

### DISCUSSION THREAD

**Discussion by PolicyPuma** [AdguardTeam/AdGuardHome#4861] [Q&A]
My domain doesn't allow me to have any subdomains so I'm forced to use path prefixes to separate my services. I'm using AdGuard Home in Docker. How do I change the path of the web interface so it would look something like this `https://my.domain/adguard`?

**Comment by ainar-g** [comment]
You can use any reverse HTTP proxy for that.  See [FAQ](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#reverseproxy).

**Reply by PolicyPuma** [reply]
> You can use any reverse HTTP proxy for that. See [FAQ](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#reverseproxy).

I've set this up however when you're not already logged in you'll be redirected from `https://my.domain/adguard` to `https://my.domain/login.html` instead of `https://my.domain/adguard/login.html`.

**Reply by ainar-g** [reply]
@PolicyPuma4, sounds like something we could improve.  Please file a bug / enhancement report about it in the Issues.  Thanks!
